web/static/js/components/idea_submission_form.jsx

```diff
@@ -19,28 +18,24 @@ const PLACEHOLDER_TEXTS = {
 
 const pushUserTypingEventThrottled = throttle((retroChannel, currentUserToken) => {
   retroChannel.push("user_typing_idea", { userToken: currentUserToken })
-}, USER_TYPING_ANIMATION_DURATION - 100)
+}, USER_TYPING_ANIMATION_DURATION - 100, { leading: true })
```

Comment:
why the need for `leading: true` here?

Reply:
Damon added it in his copy of this file. I read the documentation for it and couldn't really argue for or against it, so I figured it was harmless? I can remove it if you'd like.

Reply:
yeah, let's remove it, as it's unrelated! and thanks for the quick response!